### PR TITLE
Add function guest-info

### PR DESCRIPTION
**Pull request by XueqiangWei**
Signed-off-by: Xueqiang Wei <xuwei@redhat.com>

### CODE REVIEW

**Comment by XueqiangWei** [comment]
ID 1377084

**Comment by xutian** [comment]
is there any PR for tp-qemu depended on this PR ?

**Comment by XueqiangWei** [comment]
#752 is depended on this PR

File: virttest/guest_agent.py [hunk 1]
```diff
@@ -569,6 +569,14 @@ def get_time(self):
...
     @error_context.context_aware
+    def guest_info(self):
+        """
+        execute guest-info
+        """
+        cmd = "guest-info"
+        return self.cmd(cmd=cmd)
+
+    @error_context.context_aware
     def set_time(self, nanoseconds=None):
```
Comment (PandaWei) [inline virttest/guest_agent.py]: get_ and set_ are a couple.
Don't insert guest_info between them.
And the doc can be improved, refer to shutdown and other old functions.

Comment (XueqiangWei) [inline virttest/guest_agent.py]: got it, thx

**Comment by CongLi** [comment]
I think the commit log is not clear, virttest.guest_agent: *** is better.

**Comment by XueqiangWei** [comment]
thanks coli, will update at once

**Comment by CongLi** [comment]
LGTM,
acked-by: Cong Li <coli@redhat.com>

File: virttest/guest_agent.py [hunk 2]
```diff
@@ -581,6 +581,14 @@ def set_time(self, nanoseconds=None):
         return self.cmd(cmd=cmd, args=args)
 
     @error_context.context_aware
+    def guest_info(self):
```
Comment (xutian) [inline virttest/guest_agent.py]: why not make _get_supported_cmds call your guest_info function directly?

Comment (XueqiangWei) [inline virttest/guest_agent.py]: In check-memory-leak test case, just need to repeat send cmd "guest-info", no need to get supported cmds list. And get supported cmds covered by other test case.

Comment (xutian) [inline virttest/guest_agent.py]: I mean let '_get_supported_cmds' call your 'get_info' !

Comment (XueqiangWei) [inline virttest/guest_agent.py]: the function 'guest_info' is newly added, later than '_get_supported_cmds', I will update it

**Comment by ngu-niny** [comment]
It can be used in case qemu_guest_agent.virtio_serial.check_memory_leak without any problem, so ack the patch.

Acked-by: Nini Gu <ngu@redhat.com>

**Comment by weliao** [comment]
2016-10-20 14:17:22,069 test             L0595 INFO | PASS 1-smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.Win8.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_memory_leak

Acked-by: Wei Liao weliao@redhat.com

**Pull request merged**